tools/getmoduleversion.py

```diff
@@ -0,0 +1,19 @@
+import os,re
+
+def get_version():
+  node_version_h = os.path.join(os.path.dirname(__file__), '..', 'src',
+    'node_version.h')
+
+  f = open(node_version_h)
+
+  regex = '#define NODE_MODULE_VERSION [0-9]+'
```

Comment:
You should anchor (`^`) this to the start of the line so it won't catch e.g. a comment.